addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -12,19 +12,19 @@
                 </div>
                 <div class="col-lg-6">
                     <div class="row h-100">
-                        <div class="col-lg-6 text-center bg-alpha" data-name="Block">
+                        <div class="col-lg-6 pt24 pb8 text-center bg-alpha" data-name="Block">
                             <h3>A great title</h3>
                             <p>And a great subtitle</p>
                         </div>
-                        <div class="col-lg-6 text-center bg-200" data-name="Block">
+                        <div class="col-lg-6 pt24 pb8 text-center bg-200" data-name="Block">
                             <h3>A great title</h3>
                             <p>And a great subtitle</p>
                         </div>
-                        <div class="col-lg-6 text-center bg-200" data-name="Block">
+                        <div class="col-lg-6 pt24 pb8 text-center bg-200" data-name="Block">
                             <h3>A great title</h3>
                             <p>And a great subtitle</p>
                         </div>
-                        <div class="col-lg-6 text-center bg-beta" data-name="Block">
+                        <div class="col-lg-6 pt24 pb8 text-center bg-beta" data-name="Block">
```

Comment:
We're not supposed to do XML changes in stable version. For 14.0, let's say it is still acceptable for some days but definitely not in saas-13.X. Those are not critical bugs, can you retarget to 14.0?